src/gui/tray/UserModel.cpp

```diff
@@ -58,52 +59,43 @@ User::User(AccountStatePtr &account, const bool &isCurrent, QObject *parent)
     connect(_activityModel, &ActivityListModel::sendNotificationRequest, this, &User::slotSendNotificationRequest);
 }
 
-void User::slotBuildNotificationDisplay(const ActivityList &list)
+void User::showDesktopNotification(const QString &title, const QString &message)
 {
-    // Whether a new notification was added to the list
-    bool newNotificationShown = false;
+    ConfigFile cfg;
+    if (!cfg.optionalServerNotifications() || !isDesktopNotificationsAllowed()) {
+        return;
+    }
 
+    // after one hour, clear the gui log notification store
+    if (_guiLogTimer.elapsed() > 60 * 60 * 1000) {
```

Comment:
@FlexW Just as a nitpick, you can make this `60 * 60 * 1000` a `constexpr` variable in this function, just for it to be more readable.

Reply:
Yes that would be good :) However, it was like that before.